index.html

```diff
@@ -56,25 +62,226 @@ <h1>
             contact me.
           </p>
           <div class="social-icons">
-          <a href="" id="connect">LET'S CONNECT</a>
+            <a href="" id="connect">LET'S CONNECT</a>
             <ul id="icons">
-              <li><a href="#">
-                <img src="./images/twitter.svg" alt="twitter-logo" />
-              </a></li>
-              <li><a href="#"
-                ><img src="./images/Linkedin icon.svg" alt="linkedin-logo"
-              /></a></li>
-              <li><a href="#"><img src="./images/medium.svg" alt="medium-logo" /></a></li>
-              <li><a href="#"><img src="./images/github.svg" alt="github-logo" /></a></li>
-              <li><a href="#"
-                ><img src="./images/angelist.svg" alt="angelist-logo"
-              /></a></li>
+              <li>
+                <a href="#">
+                  <img src="./images/twitter.svg" alt="twitter-logo" />
+                </a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/Linkedin icon.svg" alt="linkedin-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/medium.svg" alt="medium-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/github.svg" alt="github-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/angelist.svg" alt="angelist-logo"
+                /></a>
+              </li>
             </ul>
           </div>
         </div>
-          
-       
       </section>
-     </header>
+
+      <section id="works">
+        <div class="card">
+          <div class="head-image">
+            <img
+              src="./images/Snapshoot Portfolio.png"
+              alt="work-image-project"
+            />
+            <h1 class="tonic">TONIC</h1>
```

Comment:
- [OPTIONAL] The previous reviewers didn't point this out so I am going to make this optional, but I strongly suggest you make these changes before merging the pull request. :1st_place_medal: You should use an `h2 or h3` tag instead of an h1. You are meant to have only 1 `h1` on your page and that should be the main title. By using appropriate heading levels, you can help search engines understand the main topics and subtopics of a page, which can improve its ranking in search results. :+1: :+1: